sell-loot.lic

```diff
@@ -82,6 +98,9 @@ class SellLoot
     return unless @bankbot_room_id
     return unless @bankbot_deposit_threshold
 
+    tip_currency = @town_data[hometown]['currency']
+    keep_copper = @bankbot_deposit_threshold*-1 if keep_copper == -1
+
```

Comment:
No problem. I was trying not to interfere with the already established methods but will give it a shot!

Thanks!